libs/@guardian/libs/src/consent-management-platform/sourcepoint.ts

```diff
@@ -56,8 +94,20 @@ export const init = (framework: ConsentFramework, pubData = {}): void => {
 	}
 
 	setCurrentFramework(framework);
+	setIsConsentOrPay(
+		isConsentOrPayCountry(countryCode) && !useNonAdvertisedList,
```

Comment:
Should this be false whenever the user is not in the CorP test?

Reply:
This would still evaluate to `true` even if `useNonAdvertisedList` is false due to not being in the test I think?

Reply:
I've updated to factor in the AB test flag.